pkg/primitives/vm/vm.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	cloudContainerFlist = "https://hub.grid.tf/tf-autobuilder/cloud-container-32b445e.flist"
+	cloudContainerFlist = "https://hub.grid.tf/tf-autobuilder/cloud-container-6fb64bb.flist"
```

Comment:
Why isn't there a latest link that we can use?

Reply:
This is to avoid overriding it by accident basically. Now we in completel control which version of zos (network) uses which version of cloud-container.